src/components/molecules/Jukebox/Jukebox.scss

```diff
@@ -0,0 +1,96 @@
+@import "scss/constants.scss";
+
+$emoji-button-diameter: $submit-button-diameter - $spacing--md;
+$jukebox-chat-max-height: 100px;
+
+.Jukebox {
+  &__container {
+    width: 100%;
+    height: 35%;
+    margin: $spacing--lg;
+  }
+
+  &__chat {
+    overflow-y: scroll;
+    display: flex;
+    flex-direction: column;
+    max-height: $jukebox-chat-max-height;
+    background-color: opaque-white(0.12);
+    height: 100%;
+    width: 100%;
+    border-top-left-radius: $border-radius--xl;
+    border-top-right-radius: $border-radius--xl;
+
+    &-messages {
+      padding: $spacing--xs $spacing--md;
+    }
+  }
+
+  &--question {
+    background-color: $dark-green;
+  }
+
+  &__form {
+    padding: $spacing--md 0;
+    display: flex;
+  }
+
+  &__inputs {
+    display: flex;
+    align-items: center;
+    place-content: center;
+    flex-grow: 1;
+    margin-right: $spacing--lg;
+  }
+
+  &__input-container {
+    flex-grow: 1;
+    display: flex;
+
+    > div {
+      width: 100%;
+      margin: auto;
+    }
+  }
+
+  &__input {
+    padding-right: $submit-button-diameter;
+    // NOTE: this removes screen artefacts of the embedded <input> inside InputField being as wide as the parent
+    height: $spacing--lg * 2;
+  }
+
+  &__buttons {
+    display: flex;
+    flex-grow: 0;
+
+    & .ButtonNG__button {
```

Comment:
can we move it out of `&__buttons` not to increase the specificity of the css rulesets?

Reply:
is it possible to move to add a class name to the ButtonNG component? So that we wouldn't need to increase the specificity of that selector?